docs/clients/adc_interface.rst

```diff
@@ -0,0 +1,115 @@
+
+.. include:: ../notes/c-full-code.rst
+
+.. code-block:: c++
+
+    #include "generic_interface.hpp"
+    #include "adc_interface_client.hpp"
+
+
+    void main(){
+        // Make a communication interface object
+        GenericInterface com;
+
+        // Make a ADC Interface object with obj_id 0
+        AdcInterfaceClient adcInterface(0);
+
+        // Use the ADC Interface Client
+        adcInterface.adc_voltage_.get(com)
+
+        // Insert code for interfacing with hardware here  
+    }
+
+Matlab
+~~~~~~
+
+To use the ADC Interface client in Matlab, all Vertiq communication code must be included in your
+path. This allows the creation of a AdcInterfaceClient object. See the Message Table below for available messages.
+All message strings use the Short Names. All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the AdcInterfaceClient is:
+
+.. code-block:: Matlab
+
+    % Make a communication interface object
+    com = MessageInterface(’COM18’,115200);
+    % Make a AdcInterfaceClient object with obj_id 0
+    adcInterface = AdcInterfaceClient(’com’,com);
+    % Use the AdcInterfaceClient object
+    adcVoltage = adcInterface.get(’adc_voltage’);
+
+Python
+~~~~~~
+
+To use the ADC Interface Client in Python, include ``iqmotion`` and create a module that has the ADC Interface Client within it's firmware. 
+See the table below for available messages. All message strings use the Short Names. 
+All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the ADC Interface Client is:
+
+.. code-block:: Python
+    :substitutions:
+    
+    import iqmotion as iq
+
+    com = iq.SerialCommunicator("/dev/ttyUSB0")
+    |variable_name| = iq.|module_name|(com, 0)
+    
+    adc_voltage = |variable_name|.get("adc_interface", "adc_voltage") 
+    print(f"adc voltage : {adc_voltage}")
+
+Message Table
+~~~~~~~~~~~~~
+
+Type ID 91 | ADC Interface
+
++--------+-------------+--------+-----------+------+-------------------------------------------------------------------------------------------------------------------------+
+| Sub ID | Short Name  | Access | Data Type | Unit | Note                                                                                                                    |
++========+=============+========+===========+======+=========================================================================================================================+
+| 0      | adc_voltage | get    | float     |      | Read only access to input analog voltage                                                                                |
++--------+-------------+--------+-----------+------+-------------------------------------------------------------------------------------------------------------------------+
+| 1      | raw_value   | get    | uint16    |      | Read only access to raw ADC value. Ex: 1V to ADC interface would return 1241 (:math:`\frac{V_{\text{in}} * 4096}{3.3}`) |
```

Comment:
this sort of looks like it is 1241 * (the fraction), can you add a colon, or something else to indicate that the equation gives back 1241